pkg/reporter/base.go

```diff
@@ -177,6 +179,35 @@ func (o RegulaOutput) AggregateByFilepath() ResultsByFilepath {
 	return byFilepath
 }
 
+type remediationRuleInfo struct {
+	URL string `yaml:"url"`
+}
+
+var remediationMap map[string]remediationRuleInfo = nil
+
+// Get the URL of the remediation documenation, or "" if no such documentation
+// exists (to our knowledge).
+func getRemediationDoc(RuleID string) string {
+	if remediationMap == nil {
```

Comment:
Maybe this should be done in an `init()` function in `rego/embedded.go`? I'm also thinking that it would be good to embed `remedation.yaml` in its own variable so that the build fails if it's not present.

Reply:
I don't really like this, mainly because it exposes the API outside of the file scope, but it's not a strong objection. I'll go ahead and make the change.